src/i_sound.c

```diff
@@ -717,19 +717,19 @@ const char **I_DeviceList(void)
 void I_BindSoundVariables(void)
 {
     M_BindNum("sfx_volume", &snd_SfxVolume, NULL, 8, 0, 15, ss_none, wad_no,
-        "Adjust sound effects volume");
+        "Sound effects volume");
     M_BindNum("music_volume", &snd_MusicVolume, NULL, 8, 0, 15, ss_none, wad_no,
-        "Adjust music volume");
+        "Music volume");
     BIND_BOOL(pitched_sounds, false,
-        "1 to enable variable pitch in sound effects (from id's original code)");
+        "Variable pitch for sound effects (from id's original code)");
```

Comment:
There is no "original id code" in here anymore. Probably "pre Doom 1.2" or similar.

Reply:
`Variable pitch for sound effects (as in Doom pre-1.2)`? I'm not sure if it's a worthwhile remark, though. I would get rid of it and leave just `Variable pitch for sound effects`. Your call.

Reply:
Yes, okay. Let's go without remark.